src/olympia/addons/utils.py

```diff
@@ -28,7 +28,12 @@ def _check(name):
         name = normalize_string(name, strip_punctuation=True).lower()
 
         for symbol in amo.MOZILLA_TRADEMARK_SYMBOLS:
-            if symbol in name:
+            violates_trademark = (
+                name.count(symbol) > 1 or (
+                    name.count(symbol) >= 1 and not
+                    name.endswith(' for {}'.format(symbol))))
```

Comment:
I'm confused, this doesn't *remove* the trademark restriction code, but patches it ?

Reply:
Yeah, looks like we need to remove some restrictions but not everything. I reverted the previous patch that added stricter checks, and removed the waffle switch that was used to enable/disable the stricter checks.